amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-config-topics.adoc

```diff
@@ -0,0 +1,166 @@
+
+. The <<queue-is-created,queue is already created>> and specified
+. The <<queue-created-by-source,queue is created>> by the source
+
+[[queue-is-created]]
+== Queue Is Already Created
+
+To correctly use a source for a queue that already exists, the user must have these actions enabled for the Amazon SQS queue:
+
+[source,json,linenums]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "sqs:DeleteMessage",
+                "sqs:GetQueueUrl",
+                "sqs:ReceiveMessage",
+                "sqs:GetQueueAttributes"
+            ],
+            "Resource": "arn:aws:sqs:REGION:ACCOUNT_ID:QUEUE_NAME"
+        }
+    ]
+}
+----
+
+Additionally, to enable the queue to receive messages when a new object is created or deleted in an S3 bucket, create a policy for the queue in this format:
+
+[source,json,linenums]
+----
+{
+  "Version": "2012-10-17",
+  "Id": "arn:aws:sqs:REGION:ACCOUNT_ID:QUEUE_NAME/SQSDefaultPolicy",
+  "Statement": [
+    {
+      "Sid": "Sid1593770403887",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "SQS:SendMessage",
+      "Resource": "arn:aws:sqs:REGION:ACCOUNT_ID:QUEUE_NAME",
+      "Condition": {
+        "ArnLike": {
+          "aws:SourceArn": "arn:aws:s3:::BUCKET_NAME"
+        }
+      }
+    }
+  ]
+}
+----
+
+[[queue-created-by-source]]
+== Queue Is Created by Source
+
+If the queue is created by the source operation, the queue name has the prefix `MULE-S3-TRIGGER`.
+
+When the queue is created by the source, the user must have permission to both create a queue and set the queue attributes:
+
+[source,json,linenums]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "sqs:DeleteMessage",
+                "sqs:ReceiveMessage",
+                "sqs:GetQueueAttributes",
+                "sqs:CreateQueue",
+                "sqs:SetQueueAttributes"
+                "sqs:GetQueueUrl"
+            ],
+            "Resource": "arn:aws:sqs:REGION:ACCOUNT_ID:MULE-S3-TRIGGER-*"
+        }
+    ]
+}
+----
+
+When the queue is created by the source operation, an SQS policy is added to the queue when the S3 object is created or deleted in the bucket in the following format:
+
+
+[source,json,linenums]
+----
+{
+  "Version": "2012-10-17",
+  "Id": "example-ID",
+  "Statement": [
+    {
+      "Sid": "example-statement-ID",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "*"
+    },
+      "Action": "SQS:SendMessage",
+      "Resource":QUEUE_ARN,
+      "Condition": {
+        "ArnLike": {
+          "aws:SourceArn": "BUCKET_ARN"
+        }
+      }
+    }
+  ]
+}
+----
+
+== S3 Policy Bucket
+
+For the S3 policy, use this format for both cases:
```

Comment:
If a user comes right to this section, the user won't know what "both cases" refers to. I suggest removing it:

Use this format for the S3 policy: